core_backend/tests/api/test_manage_content.py

```diff
@@ -114,7 +114,9 @@ def test_edit_and_retrieve_content(
 
         response = client.get(
             f"/content/{existing_content_id}",
-            headers={"Authorization": f"Bearer {readonly_token}"},
+            headers={
```

Comment:
Some of these tests are no obsolete.

Reply:
Added a ticket to remove them when we remove readonly and fullaccess references